__tests__/app.test.js

```diff
@@ -295,3 +295,74 @@ describe("POST /api/articles/:article_id/comments", () => {
     expect(response.body.msg).toBe('body text is required');
   });
 });
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+describe("GET /api/users", () => {
+
+  test("Should return with an array of users", async () => {
+    const response = await request(app)
+      .get("/api/users").expect(200)
+    const {users} = response.body
+    expect(Array.isArray(users)).toBe(true);
+  });
+
+  test("Check that user match the object structure and returns right amount of users", async () => {
+    const response = await request(app)
+      .get("/api/users").expect(200)
+    const {users} = response.body
+
+    const expected = {
+      avatar_url: "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg",
+      name: "jonny",
+      username: "butter_bridge",
+    };
+
+    expect(users[0]).toMatchObject(expected);
+    expect(users).toHaveLength(4);
```

Comment:
We are only checking one user here. We should be following the pattern of checking the shape of all of the user objects that come back and that the typeof of the property values is what we can expect. 

We can use a forEach for this 👍

Reply:
Added the forEach method to check each property value, thank you! :)